builtin/v11/init/invariants.go

```diff
@@ -63,8 +63,9 @@ func CheckStateInvariants(st *State, tree *builtin.ActorTree, actorCodes map[str
 		acc.RequireNoError(err, "unable to convert actorId %v to id address", actorId)
 		actor, found, err := tree.GetActorV5(idaddr)
 		acc.RequireNoError(err, "unable to retrieve actor with idaddr %v", idaddr)
-		acc.Require(found, "actor not found idaddr %v", idaddr)
-
+		if !found {
+			return nil // this can happen if actor self destructs as init is not informed
```

Comment:
On mainnet I've only seen Paych actors but I believe EVM actors can also self destruct.

It is hard to assert anything about these self destructed actors (like code cid) because the state itself has lost all information about the actor after self destruct.